### ISSUE TITLE
Cannot log into Wall Street Journal Online online.wsj.com

### ISSUE THREAD

**Issue by indinfer** [EFForg/privacybadgerchrome#299]
Starting conditions: Using Chrome with Privacy Badger installed and active.

1. Go to http://online.wsj.com/home-page
2. Click on the words "log in" in the upper right corner of web page. See a log in window pop up.
3. Try to log in by typing user name and password (whatever they are). Click on Log In. See that nothing happens.
4. Disable Privacy Badger for the website.
5. Try steps #2 and #3 again to log in. See that logging in works.

If possible, should try to allow log ins with Badger enabled. I understand that WE MIGHT NOT WANT to fix this if Wall Street Journal Online is violating privacy.

In any case, the workaround is to disable Badger, log in, then enable Badger. Wall Street Journal will remember that the user is logged in, even with Badger enabled.

Privacy Badger is awesome. Thank you for creating Privacy Badger.

**Comment by SwartzCr** [comment]
Interesting, privacy badger shouldn't be blocking first party requests. @indinfer could you take a screenshot of what third party trackers PB is noticing when you are at the log-in page for the wall street journal? That might help us understand what's going on

**Comment by indinfer** [comment]
The problem stopped happening. I don't know whether some software update of Chrome, Privacy Badger, or the WSJ website fixed the problem. Or maybe some configuration change I could have made on my computer fixed the problem--I don't remember any change, but then I generally don't record configuration changes. All I know is I have not experienced the problem for several weeks, or maybe months.

**Comment by SwartzCr** [comment]
@cooperq Since this is log-in on a first party site, are you sure this isn't a bug? Why would PB be blocking a session cookie from a first party source?

**Comment by indinfer** [comment]
I just tried the WSJ login again. The login still does not work with Privacy Badger enabled. However, once I log in with the "keep me logged in" box checked, I can enable Privacy Badger. Then when I open the WSJ.com page, I am still logged in.

I think in my post a day ago, I forgot that I was logged into WSJ because I used workarounds.
(1) Chrome in incognito mode allows logins.
(2) Privacy Bager disabled allows logins.
As long as I log in with "Keep me logged in" box checked, I can re-enable Privacy Badger and use Chrome in regular mode and I am still logged into WSJ.

I attach screen shots including detected tracking cookies.
![capture](https://cloud.githubusercontent.com/assets/8902149/7037945/b134d620-dd78-11e4-89c0-fdb193169b31.PNG)

![pb1](https://cloud.githubusercontent.com/assets/8902149/7038028/7d8b913c-dd79-11e4-9b16-6bc179982ff1.PNG)
![pb2](https://cloud.githubusercontent.com/assets/8902149/7038027/7d8b3458-dd79-11e4-8487-715a0c617175.PNG)
![pb3](https://cloud.githubusercontent.com/assets/8902149/7038029/7d8cceb2-dd79-11e4-97dd-ab442c5561cd.PNG)
